src/test/emulators/auth/mfa.spec.ts

```diff
@@ -462,4 +469,210 @@ describeAuthEmulator("mfa enrollment", ({ authApi, getClock }) => {
         expect(res.body.error).to.have.property("message").equals("PROJECT_DISABLED");
       });
   });
+
+  describe("when blocking functions are present", () => {
+    afterEach(async () => {
+      await updateConfig(
+        authApi(),
+        PROJECT_ID,
+        {
+          blockingFunctions: {},
+        },
+        "blockingFunctions"
+      );
+      expect(nock.isDone()).to.be.true;
+      nock.cleanAll();
+    });
+
+    it("mfaSignIn:finalize should update modifiable fields before sign in", async () => {
```

Comment:
nit: add a test that blocking function is not called if MFA is present on user but *not* performed, and `displayName` is not updated.